changelog/_unreleased/2022-07-12-cross-selling-cms-block-frontend-rendering.md

```diff
@@ -0,0 +1,36 @@
+<!--

```

Comment:
Hey,

this is the template for changelogs. https://github.com/shopware/platform/blob/trunk/changelog/_template.md

can you adjust it?

Reply:
Updated! Hope this is ok now sorry I confounded it